File: backend/src/main/java/ropold/backend/dto/EmployeeDTO.java
package ropold.backend.dto;

import ropold.backend.model.Department;

import java.util.UUID;

public record EmployeeDTO(
        UUID id,
        String personnelNumber,
        String name,
        String email,
        String phone,
        String address,
        Department department,
        boolean active,
        String notes,
        String imageUrl
) {}

File: frontend/src/components/dto/EmployeeDTO.ts
import { type Department } from "../models/EmployeeModel.ts";

export type EmployeeDTO = {
    id: string | null;
    personnelNumber: string | null;
    name: string;
    email: string | null;
    phone: string | null;
    address: string | null;
    department: Department;
    active: boolean;
    notes: string | null;
    imageUrl: string | null;
};